tests/frontend/test_tdmprogram.py

```diff
@@ -250,7 +253,7 @@ def test_one_dimensional_cluster_tokyo():
     # first half of cluster state measured in X, second half in P
     theta1 = [0] * int(n / 2) + [np.pi / 2] * int(n / 2)  # measurement angles for detector A
     theta2 = theta1  # measurement angles for detector B
-
+    sf.hbar = 2.0
```

Comment:
Hey Nico, inside the tests, it's best to do it like this:

```python
def test_one_dimensional_cluster_tokyo(monkeypatch):
    monkeypatch.setattr(sf, "hbar", 2.0)
```
`monkeypatch` is a built in pytest fixture that ensures that global attributes are only modified inside the test. This avoids the setting of `sf.hbar` from affecting other tests.

Reply:
All the tests below will also need to be updated to use monkeypatching